controllers/machinedeletionremediation_controller_test.go

```diff
@@ -456,18 +457,115 @@ var _ = Describe("Machine Deletion Remediation CR", func() {
 				})
 			})
 		})
+
+		Context("Support to MHC created CR", func() {
```

Comment:
IIUC the next two unit tests are identical except for the node name in `underTest = createRemediationOwnedByMHC(workerNode, workerNodeMachine)` (or `phantomNode`). If that was in purpose then a function for this logic might suit better instead of repeating the same code.

Reply:
Correct, but I prefer to rework the entire tests in a next PR. There is more that can be put into the same function and/or in a `DescribeTable`

Reply:
SGTM